### DISCUSSION TITLE
Redirect URL on email templates not what I'm expecting

### DISCUSSION THREAD

**Discussion by FrancoTanzarella** [supabase/supabase#22717] [Questions] [answered]
I have a specific URL added to the URL config `https://*.myapp.com/invite/success`

When i receive the email it never uses my redirect URL I specified but always uses the site URL. Is this a bug or am I missing something? I tried different tokens in the email template and all return the site URL instead of the redirect URL.

```
{{ .ConfirmationURL }} => https://myapp.com (here I'm expecting the redirect URL)
{{ .RedirectTo }} => https://myapp.com
{{ .SiteURL }} => https://myapp.com (this is expected)
```

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://github.com/supabase/supabase/assets/54564956/5d9b954e-0e38-406b-aebe-dbcd63af1a02)

Using test tool linked in Supabase docs:  https://www.digitalocean.com/community/tools/glob?comments=true&glob=http%3A%2F%2Flocalhost%3A3000%2F%2A%2A&matches=false&tests=http%3A%2F%2Flocalhost%3A3000&tests=http%3A%2F%2Flocalhost%3A3000%2F&tests=http%3A%2F%2Flocalhost%3A3000%2F%3Ftest%3Dtest&tests=http%3A%2F%2Flocalhost%3A3000%2Ftest-test%3Ftest%3Dtest&tests=http%3A%2F%2Flocalhost%3A3000%2Ftest%2Ftest%3Ftest%3Dtest

**Reply by FrancoTanzarella** [reply]
I added the following. One at a time and both at the same time. With the same results as above.

`https://*myapp.com`

**Covers**
```
- https://www.myapp.com
- https://myapp.com
```

`https://*myapp.com/**`

**Cover**
```
- https://www.myapp.com/invite/success
- https://myapp.com/invite/success
```

My results are still this
https://....supabase.co/auth/v1/verify?token=...&type=signup&redirect_to=https://myapp.com

I can't find a glob pattern that will combine the two but I don't think that will make a difference since it returns the site URL only.

**Reply by FrancoTanzarella** [reply]
Got back to this today and redirect URL is working as expected
